genesis-tools/genesis-populate/Cargo.toml

```
@@ -18,3 +18,12 @@ near-primitives = { path = "../../core/primitives" }
 near-store = { path = "../../core/store" }
 near-chain = { path = "../../chain/chain" }
 near-test-contracts = { path = "../../runtime/near-test-contracts" }
+
+[features]
+protocol_feature_flat_state=[
```

```suggestion
protocol_feature_flat_state = [
```
Formatter should have done it automatically though, did you set it correctly?